packages/playground/src/components/funds_card.vue

```diff
@@ -1,15 +1,25 @@
 <template>
-  <v-btn
-    id="tftBtn"
-    width="2000"
-    color="white"
-    @click="addTFT"
-    class="px-lg-6 px-md-2 px-sm-0 mx-sm-0"
-    style="color: white; max-width: 140px; width: auto; background-color: var(--primary)"
-    :loading="loadingAddTFT"
-  >
-    GET TFT
-  </v-btn>
+  <v-tooltip location="bottom" v-if="showTooltip()" close-delay="1000" color="primary">
```

Comment:
Do we want to remove `GET TFT` button in main/test net?

Reply:
no, we want to add tooltip to button only at qa && main net.

Reply:
@AlaaElattar Then you can disabled the tooltip not removing it
Check docs https://vuetifyjs.com/en/api/v-tooltip/#props-disabled